lib/l10n/intl_es.arb

```
@@ -544,7 +546,7 @@
     }
   },
   "cancelPendingButton": "CANCELACIÓN PENDIENTE",
-  "acceptCancelButton": "ACEPTAR CANCELACIÓN",
+  "acceptCancelButton": "CANCELAR",
```

RuboCop (1.76.1)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)